src/main/kotlin/com/neva/gradle/fork/gui/PropertyDialogField.kt

```diff
@@ -0,0 +1,60 @@
+package com.neva.gradle.fork.gui
+
+import com.neva.gradle.fork.config.properties.Property
+import com.neva.gradle.fork.config.properties.Validator
+import java.awt.Color
+import javax.swing.*
+
+class PropertyDialogField(private val property: Property, private val propField: JComponent, private val validationMessageLabel: JLabel, private val dialog: JDialog) {
```

Comment:
please reformat that to
https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting

each field in constructor on separate line

Reply:
OK